examples/gallery/line/vector-heads-tails.py

```diff
@@ -0,0 +1,78 @@
+"""
+Vector heads and tails
+----------------------
+
+Many modules in PyGMT allow to plot vectors with individual 
+heads and tails. For this purpose, several modifiers may be appended to 
+the corresponding vector-producing parameters for specifying the placement 
+of vector heads and tails, their shapes, and the justification of the vector. 
+
+To place a vector head at the beginning of the vector path 
+simply append **+b** to the vector-producing option (use **+e** to place 
+one at the end). Optionally, append **t** for a terminal line, **c** for a 
+circle, **a** for arrow (default), **i** for tail, **A** for plain open 
+arrow, and **I** for plain open tail. Further append **+l** or **+r** to 
+only draw the left or right half-sides of the selected head/tail 
```

Comment:
I feel that **+l** and **+r** is a little misleading. We can append **l** or **r**, or use **+l** or **+r**, right? The description only mentions the second case, but the example shows both.

Reply:
Yes you're right, I tried to clarify.